docker-compose.yml

```diff
@@ -68,7 +68,6 @@ relay:
     - RELAY_DOCKER_CLEAN_INTERVAL=1m
     - RELAY_COG_HOST=cog
     - RELAY_DYNAMIC_CONFIG_ROOT=/tmp/bundle_configs
-    - RELAY_MANAGED_DYNAMIC_CONFIG=true
```

Comment:
We also need to update the Cog Book bits on this to reflect that this is now the default, as well as that a directory is now basically required for the default.